dist/Renderer.js

```diff
@@ -0,0 +1,94 @@
+class Renderer{
+    generateMatrix(row, col, board){
+        let numberOfCoins = 0
+        let playerSizeRatio = 10/(parseInt(row)+parseInt(col))
+        this.gridBoard(row, col)
+        for (let r = 0; r < board.matrix.length; r++) {
+            for (let c = 0; c < board.matrix[r].length; c++) {
+                switch (board.matrix[r][c]) {
+                    case 'ghost':
+                        $('#board').append(`<div class='brick'></div>`)
```

Comment:
better hold each `<div>` inside a variable and then appends it here in the `case`s - that way it would be more readable and also you could change the `<div>`s if and when you'd like to more easily